distributed/deploy/cluster.py

```diff
@@ -71,6 +71,7 @@ def __init__(
         scheduler_sync_interval=1,
     ):
         self._loop_runner = LoopRunner(loop=loop, asynchronous=asynchronous)
+        self.__asynchronous = asynchronous
```

Comment:
I'm slightly confused. The `SyncMethodMixin` is checking for `_asynchronous` but you chose to use an attribute with name mangling (double underscore) so this attribute will never be detected by the mixin. is this intentional?

Reply:
All good. Just wanted to make sure this is intentional. For this usecase the mangling is a good choice.